packages/message-pickup-repository-dynamodb/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@credo-ts/message-pickup-repository-dynamodb",
```

Comment:
What about `didcomm-message-pickup-dynamodb` still quite long. Maybe `didcomm-pickup-dynamodb`? 

Since it's under the main credo-ts scope it's probably good to make it didcomm specific